src/main/java/uk/gov/justice/hmpps/prison/api/resource/AgencyResource.java

```diff
@@ -252,6 +256,7 @@ public List<Agency> getAgenciesByCaseload(@PathVariable("caseload") @Parameter(r
         @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
     @Operation(summary = "List of prison contact details.", description = "List of prison contact details.")
     @GetMapping("/prison")
+    @SlowReportQuery
```

Comment:
surprising this is slow

Reply:
Its only because all these are > 100ms which isnt *that slow. Im erring on the side of caution / status quo to start with. If the primary load is fine I can remove some of these (esp those based on a single offender / booking )